apps/site/lib/site_web/templates/trip_plan/_itinerary_row_step.html.eex

```diff
@@ -1,7 +1,7 @@
 <div class="route-branch-stop-list route-branch-stop <%= [@mode_class] %>">
   <%= SiteWeb.PartialView.StopBubbles.render_stop_bubbles(@bubble_params) %>
   <div class="itinerary-step">
-  <%= @step %>
+  <span class="notranslate"><%= @step %></span>
```

Comment:
This one might be too heavy-handed as it prevents the entire step's description from being translated. After poking around a bit I can't figure a way to reliably find the portion of text that contains a stop name, so let's revert this change.

<img width="649" alt="image" src="https://github.com/mbta/dotcom/assets/2136286/8456a154-848d-4585-8f8c-cab72242e950">

Reply:
@thecristen I asked Paul and he said to not translate it for now. I added that detail to the Asana task.